api/src/main/java/org/openmrs/BaseCustomizableData.java

```diff
@@ -26,7 +26,7 @@
  */
 public abstract class BaseCustomizableData<A extends Attribute> extends BaseChangeableOpenmrsData implements Customizable<A> {
 	
-	private Set<A> attributes = new LinkedHashSet<>();
+	private Set<A> attributes = new TreeSet<>();
```

Comment:
You'll also need to switch the import above from `import java.util.LinkedHashSet` to `import java.util.TreeSet`